Add alibi anchors

khellan opened the pull request:
Use alibi to generate anchors for explainability.

khellan committed:
Add alibi anchors

Use alibi to generate anchors for explainability.

haraldvinje commented:
I did not understand this block. If False is never executed?
if False:
    !pip install --upgrade torch
    !pip install --upgrade alibi
    !pip uninstall -y tensorflow-gpu
    !pip install --upgrade tensorflow

boto3 is not used and not a common package. I suggest either removing import or adding to Pipfile.

Otherwise good!

khellan commented:
> I did not understand this block. If False is never executed?
> if False:
> !pip install --upgrade torch
> !pip install --upgrade alibi
> !pip uninstall -y tensorflow-gpu
> !pip install --upgrade tensorflow
> 
> boto3 is not used and not a common package. I suggest either removing import or adding to Pipfile.
> 
> Otherwise good!

boto3 was a leftover from trying to run this in Sagemaker and read the files from S3. Removed

The if False stuff was something that I needed once either when running in Sagemaker or Azure to get the VM correctly set up. I think it was Sagemaker so I'm removing it. Running this in Sagemaker Studio didn't work anyway. The standard VMs do not have enough memory :-/ Azure did the trick :)

khellan committed:
Clean up

Remove import of boto3 as it's not used.
Remove pip installs to prepare a VM that didn't work anyway.
